Makefile

```
@@ -39,7 +39,7 @@ test: ## Run tests
 build: ## Build sidecar binary for local target system
 	$(GO) build $(BUILD_OPTS) -v -o graylog-sidecar
 
-build-all: build-linux-armv7 build-linux build-linux32 build-windows build-windows32 build-darwin build-darwin-arm64 build-freebsd
+build-all: build-linux-armv7 build-linux-arm64 build-linux build-linux32 build-windows build-windows32 build-darwin build-darwin-arm64 build-freebsd
 
 build-linux: ## Build sidecar binary for Linux
```

How about we rename this for more clarity?
```suggestion
build-linux-amd64: ## Build sidecar binary for Linux-amd64
```

---

@mpfz0r I did a larger cleanup of the Makefile. Please check! :slightly_smiling_face: